docs/projects/fractions/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Fractions in Every Language
+layout: default
+date: 2019-10-10
+last-modified: 2019-10-10
+feature-image:
+tags: [fractions]
+authors:
+---
+
+languages like python have built-in utilities or functions for working with fractions.
+Many of these fractions functions follow a similar pattern across programming languages:
+Takes a numerator and a denomenator as an attribute.
+Perform basic arithmatic and relational operations with operator overloading.
+
+## Requirements
+
+In general, a fractions library should perform the following:
```

Comment:
Can we include some requirements that make this program executable? That way we could actually include some arithmetic or something on the command line.

Reply:
yep, i can redefine main such that it takes 3 external args ./fractions(compiled prog) operand1 operator operand2 and produces output.
EX: ./fractions "3/4" + "3/15" produces minimal p/q fraction 
output: 19/20
this way we can make it executable and easily usable fraction calculator program.